src/tests/java/picard/analysis/CollectMultipleMetricsTest.java

```diff
@@ -0,0 +1,249 @@
+package picard.analysis;
+
+import htsjdk.samtools.metrics.MetricsFile;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import picard.cmdline.CommandLineProgramTest;
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+
+/**
+ * Tests the two default "programs" that have tests in CollectMultipleMetrics
+ *
+ *
+ * @author Yossi farjoun
+ */
+
+public class CollectMultipleMetricsTest extends CommandLineProgramTest {
+
+    private static final File TEST_DATA_DIR = new File("testdata/picard/sam");
+
+    public String getCommandLineProgramName() {
+        return CollectMultipleMetrics.class.getSimpleName();
+    }
+
+
+    @Test
+    public void testAlignmentSummaryViaMultipleMetrics() throws IOException {
+        final File input = new File(TEST_DATA_DIR, "summary_alignment_stats_test.sam");
+        final File reference = new File(TEST_DATA_DIR, "summary_alignment_stats_test.fasta");
+        final File outfile   = File.createTempFile("alignmentMetrics", ".txt");
+        outfile.deleteOnExit();
+        final String[] args = new String[] {
+                "INPUT="  + input.getAbsolutePath(),
+                "OUTPUT=" + outfile.getAbsolutePath(),
+                "REFERENCE_SEQUENCE=" + reference.getAbsolutePath(),
+                "PROGRAM=null",
+                "PROGRAM="+CollectMultipleMetrics.Program.CollectAlignmentSummaryMetrics.name(),
+                "PROGRAM="+CollectMultipleMetrics.Program.CollectInsertSizeMetrics.name()
+        };
+        Assert.assertEquals(runPicardCommandLine(args), 0);
+
+        final MetricsFile<AlignmentSummaryMetrics, Comparable<?>> output = new MetricsFile<AlignmentSummaryMetrics, Comparable<?>>();
+        output.read(new FileReader(outfile));
+
+        for (final AlignmentSummaryMetrics metrics : output.getMetrics()) {
```

Comment:
not really. the input files are different. it's true that I run the same two programs in both tests, but I only look at the result of the relevant one in each. the reason I run two programs is to make sure that running two programs doesn't somehow change the results.